### DISCUSSION TITLE
Can you use Phidget and TC4+

### DISCUSSION THREAD

**Discussion by PKeyes505** [artisan-roaster-scope/artisan#952] [Q&A] [answered]
Is it possible to use the Phidget for ET ant BT and use the TC4+ for Air and Heater control?

If so where do you set the Com Ports?  Right now I can get the Phidget or the TC4+ to work properly but not both at once.

![image](https://user-images.githubusercontent.com/61755459/188755719-56e34506-6750-4d6a-88cb-2ec2ba2ba8c3.png)
![image](https://user-images.githubusercontent.com/61755459/188755661-a67e8c5a-5642-453e-b4da-9e910c3b36a5.png)

**Comment by MAKOMO** [comment, marked as answer]
The simplest way is to make the TC4 the main device and the Phidgets an extra devices. Then map the Phidgets data to the ET/BT channels using a Symbolic Formula.


<img width="1056" alt="Screenshot 2022-09-07 at 09 00 08" src="https://user-images.githubusercontent.com/13909011/188811043-8633e329-997f-4724-9435-01d96f6fdfc4.png">
<img width="1056" alt="Screenshot 2022-09-07 at 09 02 01" src="https://user-images.githubusercontent.com/13909011/188811051-0dd28510-693d-435a-85bf-8c25b530bd1d.png">
<img width="1056" alt="Screenshot 2022-09-07 at 09 02 12" src="https://user-images.githubusercontent.com/13909011/188811053-4063e2e6-419c-4808-a8ba-959a68a30f62.png">

**Reply by PKeyes505** [reply]
Thank you very much... worked like a charm.
